api/tests/csv_handling_test.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+import server
+
+
+class TestCsvHandling(unittest.TestCase):
+
+    def test_path_generation(self):
+        self.assertEqual('csv/some_filename.csv',
+                         server.get_path('some_filename'))
+
+    def test_csv_exists(self):
+        with self.subTest():
+            self.assertTrue(server.csv_exists('amp_applications'))
+        with self.subTest():
+            self.assertFalse(server.csv_exists('non_existent_file'))
+
+    def test_csv_parsing(self):
```

Comment:
This test case in particular is very-very similar to one that's already in `api_test.py`. I'm thinking `test_resource_returns_partial_json`. I'm only pointing this out to underline the fact that these tests could very well belong in `api_test.py`.

The fact that they are written is commendable nevertheless!